assets/app/views/browse/route.html

```diff
@@ -0,0 +1,74 @@
+<div ng-controller="RouteController" class="content">
+  <project-page>
+    <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+    <alerts alerts="alerts"></alerts>
+    <div ng-if="route">
+      <div class="col-md-12">
+        <div class="tile">
+          <dl class="dl-horizontal left">
+            <dt>Name:</dt>
+            <dd>{{route.metadata.name}}</dd>
+            <dt>Labels:</dt>
```

Comment:
Same thing here, adding a specific todo for this.  Want to make sure annotations are treated consistently across the pages.